src/services/apiEmployees.js

```diff
@@ -0,0 +1,11 @@
+import axios from "axios";
+export async function getEployees() {
+  const { data, error } = await axios.get(
+    "https://v5bgbb7bq3.execute-api.us-east-1.amazonaws.com/development/employees"
```

Comment:
Hard-coded URL can be stored in a constant

Reply:
Fixed